src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="label__block" for="surname"> Surname:
+          <input
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+          minlength="3"
+          maxlength="32"
+          required>
+        </label>
+        <label class="label__block" for="name">Name:
+          <input
+          type="text"
+          name="name"
+          id="name"
+          autocomplete="off">
+        </label>
+        <label class="label__block" for="age">How old are You?
+          <input
+          type="number"
+          value="12"
+          id="age"
+          name="age"
+          min="1"
+          max="100">
+        </label>
+        <label class="label__block" for="birthday">Full date of birth:
+          <input
+          type="date"
+          id="birthday"
+          name="birthday">
+        </label>
+        <label class="label__block" for="terms">I accept the term of the agreement
+          <input
+          type="checkbox"
+          id="terms"
+          name="terms"
+          value="agree"
+          required>
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="label__block" for="email">E-mail:
+          <input
+          type="email"
+          id="email"
+          placeholder="email@example.com">
+        </label>
+        <label class="label__block" for="password">Password:
+          <input type="password" id="password">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="label__block">Do you love cats?
+          <label for="radio1">
+            <input
+            id="radio1"
+            type="radio"
+            name="loveCats"
+            checked
+            value="yes">Yes
+          </label>
+          <label for="radio2">
+            <input
+            id="radio2"
+            type="radio"
+            name="loveCats"
+            value="no">No
+          </label>
+        </div>
+        <label class="label__block" for="favColor">What is your favorite color?
+          <input
+          type="color"
+          id="favColor"
+          name="color">
+        </label>
+        <label for="time" class="label__block">What time do you go to bed?
+          <input
+          type="time"
+          id="time"
+          name="time">
+        </label>
+        <label class="label__block" for="favCar">What are your favorite brand of cars?
+          <select
+          name="car"
+          id="favCar"
+          required
+          multiple>
+            <option value=""
+            disabled
+            selected>
```

Comment:
```suggestion
            <option
              disabled
              selected
            >
```
Нужно ли передавтаь пустой value?